platform/ios/MGLMapView.mm

```diff
@@ -109,6 +110,7 @@ - (instancetype)initWithFrame:(CGRect)frame
     if (self && [self commonInit])
     {
         self.styleURL = nil;
+        self.accessToken = [MGLAccountManager accessToken];
```

Comment:
What about the other direction? `-[MGLMapView setAccessToken:]` should update the singleton.